eo-maven-plugin/src/test/java/org/eolang/maven/FakeMaven.java

```diff
@@ -204,6 +219,12 @@ public <T extends AbstractMojo> FakeMaven execute(final Class<T> mojo) throws IO
             "outputDir",
             this.workspace.absolute(Paths.get("target").resolve("classes")).toFile()
         );
+        final Path sodg = Paths.get("eo");
```

Comment:
@PickBas Can be inlined as it's used only in single place

Reply:
Fixed this one